content/docs/Configuration/_index.md

```diff
@@ -256,7 +256,7 @@ Please use [`get-current-version` action](/docs/actions/)
 #### actions
 
 (*string*) Custom actions/hooks overwriting the default behavior of packit
-(more in [Actions](/docs/actions/)).
+(more in [Actions](/actions/)).
```

Comment:
Will it really be on `packit.dev/actions`?

Reply:
nope, the first slash shouldn't be there /o